pkg/config/specification.go

```diff
@@ -183,16 +184,14 @@ func Load(configFile string) (*Specification, error) {
 //LoadEnv loads configuration from environment variables
 func LoadEnv() (*Specification, error) {
 	var config Specification
+	var err error
 
 	// ensure the defaults are loaded
-	if err := viper.Unmarshal(&config); err != nil {
-		return nil, err
-	}
-
-	err := envconfig.Process("", &config)
+	err = viper.Unmarshal(&config)
 	if err != nil {
-		return nil, err
+		log.WithError(err).Warn("Failed unmarshaling config")
```

Comment:
what is the point of processing config of the unmarshalling failed?

Reply:
When unmarshalling fails due to invalid config on several lines, it may still return usable configuration; but I'm still not entirely sure if it is a good idea to keep running with invalid configuration.